anki/stats.py

```diff
@@ -269,34 +283,25 @@ def _due(self, start=None, end=None, chunk=1):
     ######################################################################
 
     def introductionGraph(self):
-        if self.type == 0:
-            days = 30; chunk = 1
-        elif self.type == 1:
-            days = 52; chunk = 7
-        else:
-            days = None; chunk = 30
-        return self._introductionGraph(self._added(days, chunk),
-                               days, _("Added"))
-
-    def _introductionGraph(self, data, days, title):
+        start, days, chunk = self.get_start_end_chunk()
+        data = self._added(days, chunk)
         if not data:
             return ""
-        d = data
         conf = dict(
             xaxis=dict(tickDecimals=0, max=0.5),
-            yaxes=[dict(min=0), dict(position="right",min=0)])
+            yaxes=[dict(min=0), dict(position="right", min=0)])
         if days is not None:
             conf['xaxis']['min'] = -days+0.5
```

Comment:
The linter is complaining about this line:

```
anki/stats.py:294:35: E1130: bad operand type for unary -: NoneType (invalid-unary-operand-type)
```

but this seems both spurious and unrelated to this pull request.